### DISCUSSION TITLE
Staticgen + SSR hybrid?

### DISCUSSION THREAD

**Discussion by falseresync** [nuxt/nuxt#8495] [Questions (Nuxt 2)]
Can some pages in an SSR app be statically generated? E.g. user profile pages are SSR, while legal information is static gen

**Comment by devlsh** [comment]
@falseresync It depends on your deployment stack - e.g. are you deploying on AWS Lambda? Or on an EC2/VPS sort of system?

More information on the deployment side of things is necessary for this type of question
